box.json

```diff
@@ -22,8 +22,7 @@
             "exclude": [
                 "bin",
                 "tests",
-                "phpunit",
-                "tm"
+                "Tests"
```

Comment:
Adding `Tests` here as Symfony usually has tests installed within `src` directory. When running

```
$ composer box
```

you will notice that fewer files get installed into the PHAR.